client/components/plans/plans-compare/index.jsx

```diff
@@ -200,13 +204,13 @@ const PlansCompare = React.createClass( {
 
 		// add google-ad-credits feature
 		if ( this.isUSorCanada() && isGoogleVouchersEnabled() ) {
-			features.splice( 1, 0, googleAdCreditsFeature );
+			features.splice( 1, 0, googleAdVouchersFeature );
 		}
 		// update the description if we are also including wordpressAdCredits
 		// this is necessary here so we don't call the abtest module outside of render
 		if ( this.isUSorCanada() && isWordpressAdCreditsEnabled() ) {
-			googleAdCreditsFeature.compareDescription = googleAdCredits.getDescriptionWithWordAdsCredit();
-			googleAdCreditsFeature[ '1008' ] = '$200';
+			googleAdVouchersFeature.compareDescription = googleWordAdsAdVouchers.getDescription();
+			googleAdVouchersFeature[ '1008' ] = '$200';
```

Comment:
Agreed. However, I just refactored the code and did not write the original one. But this is an excuse, I know. :D

I don't understand what you mean by "third source"? You mean the `/plans/features` API?

Added the comment per your suggestion.